mission-control/docs/topology/concepts/connection.md

```diff
@@ -0,0 +1,65 @@
+# Connections
```

Comment:
Can we move this to references ?

Reply:
I think we also need a table for the fields

Reply:
👍🏽